ntbs-service/wwwroot/source/Components/AutocompleteSelect.ts

```diff
@@ -23,6 +22,8 @@ const AutocompleteSelect = Vue.extend({
         // Workaround for stopping Chrome trying to autocomplete by country, see discussion https://gist.github.com/niksumeiko/360164708c3b326bd1c8
         // Note that autocomplete="off" does not work.
         this.autocompleteElement.setAttribute('autocomplete', 'no');
+        // In IE, name gets automatically populated with empty string and razor then tries to bind this value to the model, so we need to remove name completely
+        this.autocompleteElement.removeAttribute('name');

```

Comment:
oh wow, that must have been fun to find!! was this what was causing the issues with particular countires?

Reply:
It just didn't work for any values on IE... and was a pain to find haha
(I checked that it looked right on IE, but not whether it would submit d'oh)